MIP/mip-1/README.md

```diff
@@ -0,0 +1,113 @@
+# MIP-1: ENTL (Enclave Nonce Time-lock)
+- **Description**: ENTL proposes a simple challenge-reponse protocol to link Enclave usage to a given program. It additionally proposes a means of time-locking to allow the replacement of said program while maintaining security.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-1](../MD/md-1)
+
+## Abstract
+
+ENTL proposes a simple challenge-reponse protocol to link Enclave usage to a given program. It additionally proposes a means of time-locking to allow the replacement of said program while maintaining security. This approach is demonstrated to be both generalizable and cryptographically sound.
+
+## Motivation
+
+This protocol was originally suggested as a solution to minimizing trust assumptions in the Atomic Bridge under [MD-1](../MD/md-1). However, similar problems were encountered under Movement Labs Governance Stage 0 concerning general key usage. 
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Protocol Description
+The protocol is a simple challenge-response scheme performed between two parties: the Enclave, a presumed secure compute unit, and the Client, the program requesting the Enclave's services. 
+
+The following messages types are exchanged between the Enclave and the Client:
+- **SYN**: the Client sends a 256-bit nonce to the Enclave.
+- **SYN-OK**: the Enclave responds indicating it has received and accepted the nonce. This can be sent as a response to either SYN or SYN-CHECK.
```

Comment:
SYN-CHECK is not described.

Reply:
Sorry, I removed this message type, but forgot to remove it here.